NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -146,7 +177,7 @@ public final class AudiobookPlayerViewController: UIViewController {
             target: self,
             action: #selector(AudiobookPlayerViewController.sleepTimerWasPressed(_:))
         )
-        sleepTimer.tintColor = self.tintColor
+        sleepTimer.tintColor = self.view.tintColor
```

Comment:
this gets system tint colors correct? great change

Reply:
Yeah I believe every tint derives from the view controller's view tint, so if a host instantiates a PlaybackViewController, then sets a custom tint on it, it's reliable results.